Mathlib/Mathport/Syntax.lean

```diff
@@ -101,13 +101,13 @@ macro ak:Term.attrKind "notation3"
   for item in lits.getArgs do
     let lit := item[0]
     if let some _ := lit.isStrLit? then
-      macroArgs := macroArgs.push (← `(macroArg| $lit:strLit))
+      macroArgs := macroArgs.push (← `(macroArg| $lit))
```

Comment:
The new name of the antiquotation kind is `str`. There probably was a reason it was applied in the first place, which would explain the failure.